testplan/exporters/testing/pdf/renderers/entries/baseUtils.py

```diff
@@ -121,11 +121,11 @@ def plot_graph(source, graph_type):
     for entry in data:
         colour = get_colour(series_options, entry)
         x_values, y_values = get_xy_coords(data[entry])
-        if graph_type is "Line":
+        if graph_type == "Line":
             plot.plot(x_values, y_values, color=colour, label=entry)
-        if graph_type is "Scatter":
+        elif graph_type == "Scatter":
             plot.scatter(x_values, y_values, color=colour, label=entry)
-        if graph_type is "Bar":
+        elif graph_type == "Bar":
             x = np.arange(len(x_values))
```

Comment:
lol @ryan-collingham my my why oh why did you ever approve this https://github.com/Morgan-Stanley/testplan/commit/37526bc66e21a36a72849b8b427de8dacaffcaed